package.json

```diff
@@ -48,13 +47,13 @@
     "types/*.d.ts"
   ],
   "scripts": {
-    "build": "yarn run release && yarn run build:es",
-    "build:es": "cross-env BABEL_ENV=es babel src --out-dir es",
+    "emitDeclarations": "tsc --skipLibCheck --emitDeclarationOnly",
+    "build": "rimraf dist && rollup -c & yarn emitDeclarations",
```

Comment:
Please remove rimraf here - it doesn't necessary

Reply:
removed rimraf